packages/server/src/restaurants/database/RestaurantSchema.ts

```diff
@@ -66,6 +67,22 @@ const RESTAURANT_CONSTANTS = Object.freeze({
   collection: 'restaurants',
 })
 class Restaurant {
+  get isCompleted() {
+    const requirements = {
+      name: this.name,
+      bankAccountNumber: this.bankAccountNumber,
+      phoneNumber: this.phoneNumber,
+      addressId: this.addressId,
+      logo: this.logo,
+      description: this.description,
+      cuisineType: this.cuisineType,
+      tags: this.tags,
+    };
+
+    if (_.values(requirements).some(_.isNil)) return false;
```

Comment:
```TS
if (wyrażenie) return true;
return false;
```
można zawsze uprościć, korzystając z tego, że i tak mamy już `wyrażenie` ([więcej info](https://adamj.eu/tech/2020/01/17/simplify-your-ifs-that-return-booleans/)).

W tym przypadku mamy na odwrót, bo robisz `false`/`true` a nie `true`/`false`, więc trzeba zanegować:
```TS
return !_.values(requirements).some(_.isNil);
```

Ewentualnie [na odwrót](https://pl.wikipedia.org/wiki/Prawa_rachunku_kwantyfikator%C3%B3w):

```TS
return _.values(requirements).every(_.negate(_.isNil));
```

Albo biorąc pod uwagę mój poprzedni komentarz:
```TS
return requirements.every(_.negate(_.isNil));
```